CHANGELOG.md

```diff
@@ -24,19 +33,19 @@
 - [Added] Two new options to the configuration relative to state tracing:
   - `targets` for specifying runtime targets to trace in WASM
   - `folder` where WASM blobs with tracing enabled are kept.
-  - More on state-tracing here: https://github.com/paritytech/substrate-archive/wiki/6.)-State-Tracing-&-Balance-Reconciliation  
+  - More on state-tracing [here](https://github.com/paritytech/substrate-archive/wiki/6.)-State-Tracing-&-Balance-Reconciliation)
```

Comment:
this looks a little funny, how does one escape `)` in markdown? :)

Reply:
> this looks a little funny, how does one escape `)` in markdown links? :)

```suggestion
  - More on state-tracing [here](https://github.com/paritytech/substrate-archive/wiki/6.\)-State-Tracing-&-Balance-Reconciliation)
```
I think ^ this works

Good catch

Reply:
I think that link actually works, but it sure looks funny.

Reply:
No, the link doesn't work, I ended up at https://github.com/paritytech/substrate-archive/wiki/6. ^